QuEST/include/QuEST.h

```diff
@@ -1662,6 +1662,41 @@ int measureWithStats(Qureg qureg, int measureQubit, qreal *outcomeProb);
  */
 Complex calcInnerProduct(Qureg bra, Qureg ket);
 
+/** Computes the Hilbert-Schmidt scalar product
+ * (which is equialent to the Frobenius inner product of matrices) 
+ * of two density matrices \p \rho_1 and \p \rho_2 of equivalent size.
```

Comment:
The Doxygen args show as `rho1` and `rho2`, so the references in the doc should be `\p rho1` (rather than `\p \rho_1` which isn't a recognised argument).

Reply:
I'll point out all the other places `\p \rho_*` should be changed below so minimise change of missing any